src/librustc_lint/lib.rs

```diff
@@ -171,6 +168,10 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
     store.register_late_pass(sess, false, true, box BuiltinCombinedModuleLateLintPass::new());
 
     late_lint_methods!(declare_combined_late_lint_pass, [BuiltinCombinedLateLintPass, [
+        // FIXME: Look into regression when this is used as a module lint
+        // May Depend on constants elsewhere
+        UnusedBrokenConst: UnusedBrokenConst,
```

Comment:
I narrow down the regression to this lint, and moving that to a non-module pass should get rid of it.